boards/arm/nrf54l15pdk_nrf54l15/doc/index.rst

```diff
@@ -0,0 +1,144 @@
++-----------+------------+----------------------+
+| TWIM      | on-chip    | i2c                  |
++-----------+------------+----------------------+
+| MPU       | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| NVIC      | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| PWM       | on-chip    | pwm                  |
++-----------+------------+----------------------+
+| GRTC      | on-chip    | counter              |
++-----------+------------+----------------------+
+| RTT       | Segger     | console              |
++-----------+------------+----------------------+
+| SPI(M/S)  | on-chip    | spi                  |
++-----------+------------+----------------------+
+| SPU       | on-chip    | system protection    |
++-----------+------------+----------------------+
+| UARTE     | on-chip    | serial               |
++-----------+------------+----------------------+
+| WDT       | on-chip    | watchdog             |
++-----------+------------+----------------------+
+
+Other hardware features have not been enabled yet for this board.
+
+Programming and Debugging
+*************************
+
+Flashing
+========
+
+To run a sample on the nRF54L15, you need to install all the
+required software.
+
+This example shows the :ref:`hello_world_user` sample,
+which uses the ``nrf54l15pdk_nrf54l15_cpuapp`` build target.
+
+To build and program the sample to the nRF54L15 PDK, complete the following steps:
+
+1. Connect the nRF54L15 PDK to you computer using the IMCU USB port on the PDK.
+#. Navigate to the :file:`zephyr/samples/hello_world` folder containing the sample.
+#. Build the sample by running the following command::
+
+      west build -b nrf54l15pdk_nrf54l15_cpuapp
+
+#. Program the sample using the standard |Zephyr| command.
+   If you have multiple Nordic Semiconductor devices, make sure that only the nRF54L15 PDK
+   you want to program is connected.
+
+   .. code-block:: console
+
+      west flash
+
+   .. note::
+
+      When programming the device, you might get an error similar to the following message::
+
+         ERROR: The operation attempted is unavailable due to readback protection in
+         ERROR: your device. Please use --recover to unlock the device.
+
+      This error occurs when readback protection is enabled.
+      To disable the readback protection, you must *recover* your device.
+
+      Enter the following command to recover the core::
+
+         west flash --recover
+
+      The ``--recover`` command erases the flash memory and then writes a small binary into
+      the recovered flash memory.
+      This binary prevents the readback protection from enabling itself again after a pin
+      reset or power cycle.
+
+Testing the LEDs and buttons in the nRF54L15 PDK
+************************************************
+
+You can test the nRF54L15 PDK with a preprogrammed :ref:`blinky-sample` sample.
```

Comment:
Preprogrammed? What? Give instructions to build it?

Reply:
Reference to blinky is wrong, must be written as:
```suggestion
You can test the nRF54L15 PDK with a preprogrammed :zephyr:code-sample:`blinky` sample.
``` 

besides the wrong reference, then I agree with @nordicjm , please remove preprogrammed, and instead describe how to build. To me, stating that board works with blinky and reference to blinky itself for build instructions is ok.

Reply:
Fixed